src/petals/client/routing/sequence_manager.py

```diff
@@ -50,7 +50,7 @@ class SequenceManagerConfig:
     ban_timeout: float = 15  # when a remote peer fails to respond, prevent routing to that peer for this many seconds
     active_adapter: Optional[str] = None  # name of active LoRA adapter (usually, Hugging Face repo)
 
-    max_pinged: int = 5  # max servers to ping from each sequence side, per update
+    max_pinged: int = 3  # max servers to ping from each sequence side, per update
```

Comment:
3 servers for start, middle, and end of sequence per update. It's stored for 5 min, so we'll have info about up to 45 servers after a while.